tools/generators/epic-generator/README.md

```diff
@@ -0,0 +1,57 @@
+# epic-generator
+
+Workspace Generator for creating a migration tracker issue epic and subsequent team sub-issues on GitHub.
+
+<!-- toc -->
+
+- [Usage](#usage)
+  - [Examples](#examples)
+- [Options](#options)
+  - [`repository`](#repository)
+  - [`title`](#title)
+  - [`message`](#message)
+
+<!-- tocstop -->
+
+## Usage
+
+```sh
+yarn nx workspace-generator epic-generator ...
+```
+
+### Examples
+
+Create an epic and sub-issues on the `microsoft/fluentui` repository with the title `Migrate vNext packages`.
+
+Leveraging positional parameters:
```

Comment:
is there any way to disable positional parameters ? generally CLI tools will use either positioning or named arguments but not both as it can be confusing

Reply:
I fail to see how it can be confusing, but I've gone ahead and removed the feature to avoid possible confusion.